nextflow_schema.json

```diff
@@ -71,34 +70,34 @@
                     "description": "Tsv indicating list of factors (conditions in the metadata table/samplesheet) and the new level on which to relevel the factor. Check contrasts docs."
                 },
                 "batch_effect": {
-                    "type": "string",
-                    "default": "false",
+                    "type": "boolean",
```

Comment:
Why has the default been removed?

Reply:
Because after the most recent template update, booleans always have a false default. You only need to add a default to the schema if it is true